docker/bootstrap/Dockerfile.mysql84

```diff
@@ -0,0 +1,23 @@
+ARG bootstrap_version
+ARG image="vitess/bootstrap:${bootstrap_version}-common"
+
+FROM --platform=linux/amd64 "${image}"
+
+USER root
+
+# Install MySQL 8.4
+RUN for i in $(seq 1 10); do apt-key adv --no-tty --recv-keys --keyserver keyserver.ubuntu.com 8C718D3B5072E1F5 && break; done && \
+    for i in $(seq 1 10); do apt-key adv --no-tty --recv-keys --keyserver keyserver.ubuntu.com A8D3785C && break; done && \
+    add-apt-repository 'deb http://repo.mysql.com/apt/debian/ bullseye mysql-8.4-lts' && \
```

Comment:
Any reason for using `mysql-8.4-lts` and not `mysql-8.0` like in the mysql80 Dockerfile?

Reply:
Yeah it seems the way mysql names it in their deb repo :/

Reply:
@L3o-pold See also https://github.com/vitessio/vitess/pull/17552, but we changed there how this is set up. We shouldn't use `add-apt-repository` (anymore) as it seems busted on Bookworm. We can instead write a source file then.